python/benchmarks/parse_pet.py

```diff
@@ -78,8 +78,8 @@ def read_metadata(self, check_quality=False) -> pd.DataFrame:
             no_index = pd.Index(names.values).difference(df.filename)
             self._data_quality_issues["missing_index"] = no_index
 
-        # TODO lance doesn't support writing booleans yet
-        with_xmls['segmented'] = with_xmls.segmented.astype(pd.Int8Dtype())
+        with_xmls['segmented'] = with_xmls.segmented.apply(
```

Comment:
`with_xmls.segments.astype('boolean')` should work (https://pandas.pydata.org/docs/user_guide/boolean.html) no?

Reply:
yup I tried, but it does not work. So the segmented is mixed with string and float NaN.